src/extension.ts

```diff
@@ -600,7 +601,7 @@ function launchMetals(
           "Test Explorer"
         );
 
-      const istTestManagerDisabled = getTestUI() === "Code Lenses";
+      const istTestManagerDisabled = !tests || getTestUI() === "Code Lenses";
```

Comment:
I don't fully get it, how is it supposed to help with issue?

Reply:
Seems that the namespace is undefined when the code is used in Theia

Reply:
ok, now I see it